package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@la-ots/pelican",
-  "version": "2.2.3",
+  "version": "2.2.4",
```

Comment:
Being a breaking change, this should be `2.3.0` most likely.

Reply:
Updated this to 2.3

Reply:
update this to 2.3.0